airship_tempest_plugin/hacking/checks.py

```diff
@@ -163,7 +163,7 @@ def no_rbac_suffix_in_test_filename(filename):
     """Check that RBAC filenames end with "_rbac" suffix.
     P101
     """
-    if "airship_tempest_plugin/tests/api" in filename:
+    if "airship_tempest_plugin/tests/api/shipyard/rbac" in filename:
```

Comment:
it shouldn't be just for shipyard as more rbac tests for other services could be added later

Reply:
ok, but maybe use regex so it is not hardcoded to only shipyard.

Reply:
this still needs to be updated. nothing here should be hardcoded to shipyard